packages/flutter_driver/lib/src/driver/web_driver.dart

```diff
@@ -181,7 +181,8 @@ class WebFlutterDriver extends FlutterDriver {
   }
 
   @override
-  Future<List<int>> screenshot() async {
+  Future<List<int>> screenshot({ScreenshotFormat format = ScreenshotFormat.png}) async {
+    assert(format == ScreenshotFormat.png, 'Web Driver only supports PNG screenshot format');
```

Comment:
I would make this an `ArgumentError` (yeah all tests use assertions but driver can be used outside of tests).